modules/node/src/transfer/transfer.service.ts

```diff
@@ -121,14 +128,15 @@ export class TransferService {
     return await this.linkedTransferRepository.findAll();
   }
 
+  // DEPRECATED
   async setRecipientAndEncryptedPreImageOnLinkedTransfer(
     senderPublicIdentifier: string,
     recipientPublicIdentifier: string,
     encryptedPreImage: string,
     linkedHash: string,
   ): Promise<LinkedTransfer> {
-    this.log.debug(
-      `setRecipientAndEncryptedPreImageOnLinkedTransfer(${senderPublicIdentifier}, ${recipientPublicIdentifier}, ${encryptedPreImage}, ${linkedHash}`,
+    this.log.warn(
+      `setRecipientAndEncryptedPreImageOnLinkedTransfer is DEPRECATED, client needs to be updated`,
```

Comment:
this should be logged client side somewhere instead

sidetone -- will this change break all our existing clients who we want to reclaim from? (i don't think so, want to make sure)

Reply:
Nope, if they have ever called this in the past, everything will be set for them in the DB.